tracker/logger.py

```diff
@@ -0,0 +1,20 @@
+"""
+This file contains the DailyLogFile observer that is used by twistd to save
+tracker logs.
+"""
+from twisted.python.log import FileLogObserver
+from twisted.python.logfile import DailyLogFile
+
+# TODO: Hard code the paths for now. Need to find a better solution.
```

Comment:
also: https://twistedmatrix.com/documents/14.0.0/core/howto/application.html#customizing-twistd-logging

I would set it like this, we can't use the default one as it would break the statistics stuff, so it's not a good idea to leave it as default.